.eslintrc

```diff
@@ -15,17 +15,27 @@
     "jsx": true
   },
   "rules": {
-    "quotes": 0,
     "no-underscore-dangle": 0,
     "curly": [2, "multi-line"],
     "camelcase": [2, {"properties": "never"}],
-    "space-in-parens": [2, "never"],
+    "space-in-parens": [2, "always"],
```

Comment:
This is the only one I see as controversial - I think we're split distribution/SB on it (using "never" in distribution code). I'm of the opinion the extra spacing adds noise rather than improves readability, others the opposite, and it's easy to find styleguides from prominent organisations recommending either option.